src/Exception/Exception.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Money\Exception;
+
+/**
+ * Base Exception.
+ *
+ * @author Frederik Bosch <f.bosch@genkgo.nl>
+ */
+
+abstract class Exception extends \Exception
```

Comment:
That way we can use SPL exceptions as well and extend whichever we want. For example UnresolvableCurrencyPairException should be invalid argument.